docs/v2/create-a-new-release.md

```diff
@@ -0,0 +1,17 @@
+# Creating a new release of ASO v2
+
+1. Go to the [releases](https://github.com/Azure/azure-service-operator/releases) page and draft a new release.
+2. In the tag dropdown, type the name of the new tag you'd like to create (it should match the pattern of previous releases tags). The release target should be `master` (the default).
```

Comment:
Include an example of the pattern here just for belt-and-suspenders?

Reply:
Done